### PR TITLE
Fix weird Card spacing

### PR DESCRIPTION
**Pull request by kokodavid**
Fix for odd spacing in spaces, related spaces , members and events cards . Also fixes the number in the +-counter.

### CODE REVIEW

**Comment by kokodavid** [comment]
image
image
image
image

**Comment by gnunicorn** [comment]
@kokodavid what does it look like in portrait/mobile mode now?

File: app/lib/features/space/pages/related_spaces_page.dart [hunk 1]
```diff
@@ -41,8 +41,8 @@ class RelatedSpacesPage extends ConsumerWidget {
           ...spaces.when(
             data: (spaces) {
               final widthCount =
-                  (MediaQuery.of(context).size.width ~/ 600).toInt();
-              const int minCount = 2;
+                  (MediaQuery.of(context).size.width ~/ 300).toInt();
+              const int minCount = 3;
```
Comment (gnunicorn) [inline app/lib/features/space/pages/related_spaces_page.dart:45]: if we are reusing the static numbers, shouldn't we put them into a central location? maybe even our custom theme?

**Comment by kokodavid** [comment]
> @kokodavid what does it look like in portrait/mobile mode now?

Sharing screenshots in a bit

**Review by gnunicorn — APPROVED** [review]
Would just like to confirm this doesn't break the mobile UI view...

**Comment by kokodavid** [comment]
@gnunicorn here are the mobile screenshots
related spaces
members
spaces view
events

**Comment by gnunicorn** [comment]
@kokodavid thanks david. now the only thing left is making the flutter style checker happy. probably a `dart fix --apply` is enough...

**Comment by kokodavid** [comment]
> @kokodavid thanks david. now the only thing left is making the flutter style checker happy. probably a `dart fix --apply` is enough...

Noted. applying the fixes in a bit.

**Review by kumarpalsinh25 — APPROVED** [review]
LGTM!

File: app/lib/features/space/pages/related_spaces_page.dart (hunk 1 above)
Comment (kokodavid) [reply app/lib/features/space/pages/related_spaces_page.dart:45]: minCount will be different for each grid like spaces and members have different minimum count.

**Comment by gnunicorn** [comment]
@kokodavid you still working on this?

**Comment by kokodavid** [comment]
> @kokodavid you still working on this?

Apologies for the late reply just forgot to mention it's done.

**Pull request merged**